docs/exploration-and-organization/content-verification.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Content verification
+---
+
+# Content verification
+
+{% include plans-blockquote.html feature="Content verification" %}
+
+![Verified icon](./images/verified-icon.png)
+
+Admins can verify items to let others know that the content is accurate and up to date with the correct data source and query.
```

Comment:
I am having a bit of trouble parsing this sentence. In my mind "up to date with" works like "I'm keeping up to date with API changes", but that doesn't match the rest of the sentence, I think? Maybe a comma before with? or something like `..and uses the correct data sources...`?